.clang-format

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
While I approve, this PR is about CMake. This is not a CMake related change.

Reply:
That is true, but I love it!